syz-manager/rpc.go

```diff
@@ -269,7 +269,7 @@ func (serv *RPCServer) NewInput(a *rpctype.NewInputArgs, r *int) error {
 
 	f := serv.fuzzers[a.Name]
 	if f != nil {
-		f.instModules.Canonicalize(a.Cover, a.Signal)
+		f.instModules.Canonicalize(&a.Cover, &a.Signal)
```

Comment:
If we drop all signal from the input, will it be saved to the corpus and shared with other `syz-fuzzer`s? Should it?

Reply:
Good question. Currently the input is still saved to the corpus and shared with other fuzzers. If all signals are dropped, it implies that all coverage belongs to a module which may not exist on other fuzzers. However, if we're assuming that each fuzzer is running the same kernel boot, the module would likely exist at some point and the input would be a valid program to fuzz. I think it may be good to keep the input but that's just a guess; if you think it would be better to drop it as the input may not be valid I can drop it as well